packages/yourloops/components/dashboard-widgets/medical-files/medical-report-list.tsx

```diff
@@ -109,7 +110,8 @@ const MedicalReportList: FunctionComponent<CategoryProps> = (props) => {
   }
 
   useEffect(() => {
-    if (!medicalReports) {
+    if (!medicalReports && shouldMakeInitialApiCallToFetchMedicalReports.current) {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏What do you think of `isDataFetched` as a generic name that we could reuse in all components/hooks requiring to fetch data on mount?

This would require to also do the change in
- patient hook
- team hook

Reply:
`isDataFetched` would mean that `isDataFetched` could be set to `true` but we would still have cases where we would fetch the data again. With a naming such as `isDataFetched` one could easily think that you don't fetch any data when this is true